lib/commands/general.js

```diff
@@ -116,6 +116,12 @@ commands.keys = async function (keys) {
     }
     await this.setValue(keys, el.ELEMENT);
   } else {
+    if (_.isArray(keys)) {
+      keys = keys.join('');
+    }
+    if (!_.isString(keys)) {
+      keys = `${keys}`;
```

Comment:
Hmm, I was more commenting on what we would prefer to standardize on if

    `${keys}`

and

    keys.toString()

are equivalent.